milli/src/criterion.rs

```diff
@@ -63,3 +64,20 @@ pub fn default_criteria() -> Vec<Criterion> {
         Criterion::Exactness,
     ]
 }
+
+impl fmt::Display for Criterion {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        use Criterion::*;
+
+        match self {
+            Typo            => write!(f, "typo"),
+            Words           => write!(f, "words"),
+            Proximity       => write!(f, "proximity"),
+            Attribute       => write!(f, "attribute"),
+            WordsPosition   => write!(f, "wordsPosition"),
+            Exactness       => write!(f, "exactness"),
+            Asc(attr)       => write!(f, "asc({:?})", attr),
+            Desc(attr)      => write!(f, "desc({:?})", attr),
```

Comment:
```suggestion
            Typo            => f.write_str("typo"),
            Words           => f.write_str("words"),
            Proximity       => f.write_str("proximity"),
            Attribute       => f.write_str("attribute"),
            WordsPosition   => f.write_str("wordsPosition"),
            Exactness       => f.write_str("exactness"),
            Asc(attr)       => write!(f, "asc({:?})", attr),
            Desc(attr)      => write!(f, "desc({:?})", attr),
```

Reply:
I don't mind changing, but why? I find it somewhat nicer when they are all the same :)

Reply:
Just to skip the formatting part, as it is not necessary :)